test/parse.jl

```diff
@@ -878,3 +878,8 @@ let f = function (x; kw...)
     @test f(1) == (1, Any[])
     @test g(1) == (1, 2)
 end
+
+# issue #8925
+global const (c8925, d8925) = (3, 4)
```

Comment:
Perhaps not necessary, but you could simultaneously test the `global`ness by wrapping the definition in a `let` block and leaving the `@test`s outside.